packages/gestalt/src/Flyout/__tests__/__snapshots__/Flyout.jsdom.test.js.snap

```diff
@@ -9,3 +9,11 @@ exports[`Flyout renders 1`] = `
   <div />
 </div>
 `;
+
+exports[`Flyout renders as error 1`] = `
```

Comment:
Hmm this doesn't seem right.

Reply:
Good catch, the `Flyout` `Controller` child render depends on a ref being existent in order to fully render so the test was treating it as not being set. By using the `createNodeMock` option to some random truthy value we can bypass that and get the full render going.